### ISSUE TITLE
Description can be improved

### ISSUE THREAD

**Issue by luismanson** [typeling1578/vertical-tabs#3]
Hello, this extension looks really nice! It was my first option to install. 
I hope you don't mind suggestion. I had to also search for Tab Center Reborn and go read their site to see any information on the extension, what it does and it's features. It would be nice to have a better description of the extension in overall.

I'm also wondering, can I move the tab into the left side? I tried with CSS but it's not move the tab itself.

TLDR: It would be nice to have a good description in the the README and/or Mozillas's site.

Thanks!
